blog/2025-08-30-K3s-1.34-release.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Kubernetes v1.34 is out!
+description: Step back and look at K3s recent work and achievements
+authors: manuelbuil # Fill out your info in the authors.yml file
+hide_table_of_contents: true
+---
+Kubernetes 1.34 is finally here! A look back at K3s recent work and achievements 🥳
+
+It's been a busy and productive few months, and we're excited to share some of the amazing progress we've made. From new features that make your lives easier to important bug fixes and foundational improvements, our team and community have been hard at work.
+
+## Key Features and Improvements ✨
+Here’s a look at some of the most impactful features and updates we've rolled out:
+
+#### Automatic Certificate Renewal Window Increase 📅
+We've made certificate management more user-friendly. We increased the automatic certificate renewal window from 90 to 120 days. This means if you perform quarterly upgrades, your certificates will be renewed more frequently, preventing them from expiring and making your clusters unusable. For more information check our [docs](https://docs.k3s.io/cli/certificate#client-and-server-certificates)
+
+#### Optional Airgap Image Tarball Imports 💨
+For those who use K3s in an airgap environment, you now have the option to skip importing all image tarballs. You could select to import only images that have changed since they were last imported, even across restarts. This can significantly speed up the startup process, as the kubelet can start sooner. Check our [docs](https://docs.k3s.io/installation/airgap?airgap-load-images=Manually+Deploy+Images#enable-conditional-image-imports) for more information
```

Comment:
```suggestion
For those who use K3s in an airgap environment, you now have the option to skip importing all image tarballs. You can select to import only images that have changed since they were last imported, even across restarts.  This can significantly speed up the startup process when deploying a large number of images, as the kubelet can start sooner. Check our [docs](https://docs.k3s.io/installation/airgap?airgap-load-images=Manually+Deploy+Images#enable-conditional-image-imports) for more information
```